src/api/todos.ts

```diff
@@ -0,0 +1,19 @@
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+
+export const getTodos = (userId: number) => {
+  return client.get<Todo[]>(`/todos?userId=${userId}`);
+};
+
+// Add more methods here
+export const addTodos = (userId: number, todo: {
+  userId: number;
+  title: string;
+  completed: boolean;
+}) => {
```

Comment:
small improvement for the future. you can use you `Todo` typing
```suggestion
export const addTodos = (userId: number, todo: Omit<Todo, 'id'>) => {
```